contents/docs/user-guides/organizations-and-projects.md

```diff
@@ -91,3 +94,5 @@ With **Per-project access** enabled, nothing changes for organization members at
 Access to each project can then be granted individually only for members who need it, in project settings.
 
 In this mode you can also grant members project-specific access level that is _above_ their organization-wide one. In such case they will have privileges of the higher level, but only for that specific project.
+
+> **Note:** Per-project access belongs to our premium team-oriented offering. To use this feature, [set up PostHog Cloud billing](https://posthog.com/pricing) or [contact us](https://share.hsforms.com/1-IVCY9gNRvaZBajMt_UPIg4559u) for a self-hosted license.
```

Comment:
I think it's better to take the user to the pricing page either way, in case we letter change the form this doesn't get outdated.

Reply:
One additional thing. We don't specify how to enable/disable per-project access.

Reply:
Make sense re: first point. Though that form link is still in a ton of places on the website.
Re: second point, we did specify a few paragraphs above, but I made it stand out more.